pkg/client/registry.go

```diff
@@ -6,6 +6,27 @@ import (
 	clusterC "github.com/spectrocloud/hapi/spectrocluster/client/v1"
 )
 
+func (h *V1Client) GetRegistryPackByName(registryName string) (*models.V1PackRegistry, error) {
```

Comment:
Please change method name to `GetPackRegistryByName`

Reply:
Done Rishi